frame/session/src/lib.rs

```diff
@@ -703,7 +703,7 @@ impl<T: Config> Pallet<T> {
 		<QueuedChanged<T>>::put(next_changed);
 
 		// Record that this happened.
-		Self::deposit_event(Event::NewSession(session_index));
+		Self::deposit_event(Event::NewSession { index: session_index });
```

Comment:
You can then do the following after the rename:

```suggestion
		Self::deposit_event(Event::NewSession { session_index });
```